src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -14134,6 +14136,12 @@ def _window_agg(
                 ErrorMessage.not_implemented(  # pragma: no cover
                     f"Window aggregation does not support the aggregation {repr_aggregate_function(agg_func, agg_kwargs)}"
                 )
+            if (
+                snowflake_agg_func.snowpark_aggregation is not count
```

Comment:
Can you add a comment saying that `count` does not output Timedelta columns which is why it is supported?

Reply:
Done.